src/gpu/intel/jit/v2/conv/kernel_desc_2d_reqs.cpp

```diff
@@ -145,6 +145,49 @@ block_2d_params_t to_block_2d_params(const prop_kind_t &prop,
     return params;
 }
 
+void generate_2d_stride_var_reqs(layout_tag_t &tag, stride_t &P,
+        prb_reqs_t &reqs, tensor_kind_t &tensor_kind, const kernel_desc_t &desc,
```

Comment:
```suggestion
        prb_reqs_t &reqs, const tensor_kind_t &tensor_kind, const kernel_desc_t &desc,
```

Please double check where we can add `const`